composer.json

```diff
@@ -20,8 +20,8 @@
     ],
     "require": {
         "php": "^7.0",
-        "container-interop/container-interop": "^1.1",
-        "psr/log": "^1.0"
+        "psr/log": "^1.0",
+        "psr/container": "^1.0"
```

Comment:
@dutekvejin Please sort these two alphabetically (priority also), so that `psr/container` is above `psr/log`.